plugins/node/opentelemetry-instrumentation-express/test/express.test.ts

```diff
@@ -620,20 +620,23 @@ describe('ExpressInstrumentation', () => {
         //     `- span 'middleware - simpleMiddleware'
         //     `- span 'request handler - /post/:id'
         const spans = collector.sortedSpans;
-        assert.strictEqual(spans[0].name, 'GET /post/:id');
-        assert.strictEqual(spans[0].kind, SpanKind.CLIENT);
-        assert.strictEqual(spans[1].name, 'middleware - query');
-        assert.strictEqual(spans[1].kind, SpanKind.SERVER);
+        assert.strictEqual(spans[0].name, 'GET');
+        assert.strictEqual(spans[0].kind, testUtils.OtlpSpanKind.CLIENT);
```

Comment:
reviewer note: with https://github.com/open-telemetry/opentelemetry-js/pull/4866 http/https client instrumentation was fixed and now produces a client span.

Reply:
Oh wow, this makes so much more sense. Thanks for fixing this!

Reply:
Thanks for chasing this down, Marc!

Sorry for the confusion.
FWIW, the `testUtils.runTestFixture()` stuff was based on a similar thing we have at work on a separate repo. In that case the equivalent of the TestCollector class **normalizes the span objects somewhat**. One of those normalizations is to translate the OTLP `span.kind` and `span.status` values to their enum strings. So, test asserts are doing `assertEqual(span.kind, 'SPAN_KIND_CLIENT')` and `assertEqual(span.status.code, 'STATUS_CODE_ERROR')`.

Reply:
Again, FWIW, here is the normalization code I mean: https://github.com/elastic/elastic-otel-node/blob/main/packages/mockotlpserver/lib/normalize.js#L114-L154
I'm not proposing we use this in this repo.
I'd vaguely recall seeing something about a canonical JSON representation of OTLP service requests or of OTel signals. Given that, I think that would be a good target for normalization, and then tests could assert against that normalized form.

Reply:
https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2421 to get one more occurrence of `SpanKind.*` usage in this test file.